### DISCUSSION TITLE
Milvus Java SDK help (https://github.com/milvus-io/milvus-sdk-java) needed.

### DISCUSSION THREAD

**Discussion by samjax** [milvus-io/milvus#34761] [Q&A and General discussion]
I am trying to understand the milvus java sdk's (https://github.com/milvus-io/milvus-sdk-java) responses when it comes to resources (CPU and Memory) used by different components((ex. IndexCoord or Datanode) in a Milvus cluster . I am trying to get to the part of the code in the sdk where this is being handled. Reason is, when I run a "kubectl top" on the milvus components running in my milvus cluster deployed on AWS, I see the values I get for cpu and memory usage are much smaller (by a magnitude of 10) compared to what is being returned by the sdk. I am not a java developer so I am not proficient enough to track down the code that does this. Can someone please help? 

Thanks a lot.

**Comment by xiaofan-luan** [comment]
what fucntion did you call and what result did you get?

**Comment by samjax** [comment]
I was calling the getMetrics() method.

**Comment by xiaofan-luan** [comment]
I think the GetMetrics doesnt contain any cpu usage info?
Best way to get that is using grafana and prometheus from milvus dashboard

**Comment by samjax** [comment]
I will try the grafana dashboards, but what is the metrics the sdk is sending back when getMetrics() method is called upon?
